SrfGen/NHM/nhm2srf.py

```diff
@@ -16,11 +15,17 @@
 from createSRF import leonard, skarlatoudis, CreateSRF_multi, gen_meta
 from qcore import geo, simulation_structure
 
-FILE_LOCATION = os.path.dirname(os.path.abspath(__file__))
+SCRIPT_LOCATION = os.path.dirname(os.path.abspath(__file__))
 
 MAGNITUDE_ROUNDING_THRESHOLD = 7.5
 
 
+def get_seed():
+    """Creates a seed for genslip.
+    This number is the maximum value of a 32 bit int"""
+    return random.randint(2147483647)
```

Comment:
Is there a python constant rather than using the number?

Comment should state limitation of genslip code aka why this number.